Resources/Prototypes/Entities/Mobs/NPCs/revenant.yml

```diff
@@ -58,6 +58,17 @@
     rules: ghost-role-information-revenant-rules
   - type: GhostTakeoverAvailable
   - type: Revenant
+    malfunctionWhitelist:
```

Comment:
Why not just call this emag instead of malfunction? 
Would just be clearer.

Additionally, what about creating a block list alongside the allow list.
With the allow list working as a "not empty then only specified" list.

Reply:
the ability is called malfunction and i guess a blacklist wouldnt hurt